src/GenericAccessRequestForm/Constants/FormSpecificConstants.ts

```diff
@@ -0,0 +1,50 @@
+const StudyRegistrationAccessRequest = {
+  name: 'StudyRegistration',
+  title: 'Study Registration Access Request',
+  description:
+    'Please fill out this form to request and be approved for access to register your study with the HEAL Platform.',
+  resultTitle: 'Your access request has been submitted!',
+  resultSubtitle:
+    'Thank you for your submission. Requests take up to 1 business day to complete. You will be notified of the status.',
+  showStudyName: true,
+  showGrantNumber: false,
+  showDisclaimer: true,
+  subjectLine: 'Study registration access request for',
+};
+const DataDictionarySubmissionRequest = {
+  name: 'DataDictionarySubmissionRequest',
+  title: 'Data Dictionary Submission Request',
+  description:
+    'Please fill out this form to request access to submit data dictionaries to your study on the HEAL Platform.',
+  resultTitle: 'Your access request has been submitted!',
+  resultSubtitle:
+    'Thank you for your submission. Requests take up to 1 business day to complete. You will be notified of the status.',
+  showStudyName: true,
+  showGrantNumber: false,
+  showDisclaimer: true,
+  subjectLine: 'Data dictionary submission access request for',
+};
+const WorkSpaceRegister = {
+  name: 'WorkSpaceRegister',
+  title: 'Workspace Registration Access Request',
+  description: `The HEAL workspace provides a standardized computational environment for accessing and analyzing data from HEAL studies.
```

Comment:
For the workspace access request form this needs to be configurable from portal config. For example BRH won't want to have an access form mentioning HEAL on it

Reply:
NOTES from discussion with Mingfei Apr. 20th

See line 171 in uc-cdis/data-portal/src/WorkspaceRegistration/WorkspaceRegistration.tsx in Master branch
```    const infoMessage = workspaceRegistrationConfig?.workspaceInfoMessage || 'Please fill out this form to request and be approved for access to workspace.';```

Add to FormSpecificConstants import 
```
import {
workspaceRegistrationConfig
} from '../localconf';
```

And have description determined based on this conditional for Workspace Registion

```
description:  workspaceRegistrationConfig?.workspaceInfoMessage || 'Please fill out this form to request and be approved for access to workspace.

Reply:
Per discussion, the description value for the WorkspaceAccessRequest object in FormSpecificConstants.ts has been updated to 

```  description: workspaceRegistrationConfig?.workspaceInfoMessage || 'Please fill out this form to request and be approved for access to workspace.',```

And the workspaceRegistrationConfig has been imported from the localconf file in FormSpecificConstants.ts